packages/components/search/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
use same eslint as kronos -

Reply:
I will do some more work  on eslint for exact like kronos.